zeebe/engine/src/main/java/io/camunda/zeebe/engine/processing/tenant/TenantUpdateProcessor.java

```diff
@@ -104,7 +104,7 @@ private void updateExistingTenant(
       existingTenant.setName(updatedName);
     }
     final var updatedDescription = updateRecord.getDescription();
-    if (!updatedDescription.isEmpty()) {
+    if (!existingTenant.getDescription().equals(updatedDescription)) {
```

Comment:
❓ Isn't the purpose of these `isEmpty` checks to enable partial updates, e.g. only setting the description and leaving the name unchanged?

Reply:
I noticed an issue with the isEmpty checks blocking the removal of a description (i.e. if I set it to `""` it keeps the old description which felt counter intuitive.

Reply:
We decided against supporting partial updates. Both name and description have to be provided when updating a tenant.

Reply:
If we don't do partial updates, can we remove all the `isEmpty()` checks in the update processors?

Reply:
One more question: can we set the description unconditionally or is there a side effect when we call the setter?

Reply:
I'm only seeing one `isEmpty` check for the name which has to stay because we don't want a blank name. Do you mean other `isEmpty` checks than that?

Reply:
There are several in the [other update processors](https://github.com/camunda/camunda/blob/59e3d72b4253852f3075851bd166a923c05bc851/zeebe/engine/src/main/java/io/camunda/zeebe/engine/processing/user/UserUpdateProcessor.java#L104).
 If we don't support partial updates, wouldn't validation catch an empty name? The way it works now is that if a full update with an empty name is processed we silently ignore the name, although this indicates a problem upstream.

Reply:
I see. The decision to not support partial updates was made only for tenants, not necessarily the other entities.

I agree that ignoring a blank name here is a bit weird :+1: